src/caches/sdirk_caches.jl

```diff
@@ -611,7 +603,7 @@ mutable struct Kvaerno3Cache{uType,rateType,J,JC,UF,uEltypeNoUnits,tType,Tab} <:
   tab::Tab
 end
 
-u_cache(c::Kvaerno3Cache)    = (c.uprev2,c.zᵧ,c.z,c.Δzᵧ,c.Δz)
+u_cache(c::Kvaerno3Cache)    = (c.z₁,c.z₂,c.z₃,c.z₄,c.dz)
```

Comment:
no, but that needs a lot more to work. It'll have to be another PR which extends the cache tests: these are very difficult on the stiff methods.